src/main/java/com/iota/iri/network/Neighbor.java

```diff
@@ -84,19 +85,27 @@ void incRandomTransactionRequests() {
     public void incInvalidTransactions() {
     	numberOfInvalidTransactions++;
     }
-    
+
+    void incStaleTransactions() {
```

Comment:
I suppose you didn't add `public` like the rest of the methods becuase you only want classes in the network package to have access.

I am fine with that :-)